lib/phone_shop/widgets/components/text_input.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+
+class TextInput extends StatefulWidget {
+  final bool isDisabled;
```

Comment:
if in original TextField have param enabled do not change that name because programmers used to it.